stdlib/websub/src/main/ballerina/src/websub/subscriber_service_endpoint.bal

```diff
@@ -35,6 +35,10 @@ public type Listener object {
 
     private http:Listener? serviceEndpoint = ();
 
+    # The initialization method for the Listener.
+    #
+    # + port - The port to start listening on
```

Comment:
```suggestion
    # + port - The port to listen
```

Reply:
IMO "The port to listen" is incomplete. 

We can maybe say "The port to listen on"

@chamil321 thoughts?

Reply:
Since we listen "to" something, "port to listen" is correct.

Reply:
In the context of a Server, "The port to listen on" would make sense

Reply:
+1